src/learn/augur-to-auspice.rst

```diff
@@ -206,6 +213,100 @@ If a coloring was provided for this key then Auspice will attempt to display
 the title rather than the key. Selecting filters here behaves the same was as
 sidebar data filtering (:ref:`see [a2], above <auspice-component-sidebar-filter>`).
 
+.. _auspice-component-diversity-panel:
+
+Diversity (Entropy) Panel
+---------------------------------
+
+We'll use the following annotated screenshot of the diversity (entropy) panel
+of the  `measles virus tree <https://nextstrain.org/measles>`__ (built using
+`this Augur pipeline <https://github.com/nextstrain/measles>`__) to introduce the
+various components:
+
+.. image:: ../images/auspice-components-diversity-panel.png
+  :alt: Annotated screenshot of Auspice's diversity (entropy) panel
+
+The diversity panel is enabled by data in the :term:`dataset JSON <dataset>`.
+The top-level ``meta.genome_annotations`` provides the genome annotations
+displayed and the individual tree nodes provide the mutations
+via ``node.branch_attrs.mutations``, which are used to calculate the entropy
+and to count the mutation events.
+
+Gaps (``-``), masked nucleotides (``N``), and unknown amino acids (``X``) are
+excluded from the calculations and counts. Only mutations from visible tree
+nodes are included in the calculations for the diversity panel, so the entropy values
+and event counts will change when you turn on a filter or zoom into a subtree.
+
+.. _auspice-component-diversity-panel-entropy-event-toggle:
+
+[b1] Toggle between Entropy and Events
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:guilabel:`ENTROPY` represents normalized `Shannon entropy <https://en.wikipedia.org/wiki/Entropy_\(information_theory\)>`__,
+measuring the "uncertainty" inherent in the possible nucleotides or codons
+at a given position.
+
+:guilabel:`EVENTS` represent a count of changes in the nucleotide or codon at that
+position across the visible tree.
```

Comment:
Might be worth adding that both mutations to and from the excluded characters (`-`, `N` for nuc; `X` for aa) are ignored here.